dora/src/bytecode/data.rs

```diff
@@ -380,6 +401,8 @@ impl BytecodeOpcode {
             | BytecodeOpcode::InvokeStaticPtr
             | BytecodeOpcode::NewObject
             | BytecodeOpcode::NewArray
```

Comment:
All the LoadArrayXXX and StoreArrayXXX also need the position

Reply:
Done.